### PR TITLE
tests: fix rawkv build

### PR DESCRIPTION
**Pull request by disksing**
Signed-off-by: disksing <i@disksing.com>

### What problem does this PR solve? 
rawkv is currently moved to a standalone package and some methods are changed with an extra context parameter.

### What is changed and how it works?
update rawkv usage.

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
```
cp go.mod1 go.mod
cp go.sum1 go.sum
go build ./...
```

### Release note
- No release note

### CODE REVIEW

File: tests/br_rawkv/client.go [hunk 1]
```diff
@@ -32,8 +33,8 @@ var (
 		"kv pairs are separated by commas, key and value in a pair are separated by a colon")
 )
 
-func createClient(addr string) (*tikv.RawKVClient, error) {
-	cli, err := tikv.NewRawKVClient([]string{addr}, config.Security{
+func createClient(addr string) (*rawkv.Client, error) {
+	cli, err := rawkv.NewClient(context.TODO(), []string{addr}, config.Security{
```
Comment (kennytm) [inline tests/br_rawkv/client.go:37]: using `context.TODO()` means this will change to accept a generic context in future versions?

Comment (disksing) [reply tests/br_rawkv/client.go:37]: Yes, if in the future we want to apply context to our tests, a context can be passed from the upper level to replace `context.TODO()`.

**Comment by disksing** [comment]
Please take a look. @kennytm @overvenus

**Review by YuJuncen — APPROVED** [review]

**Review by kennytm — APPROVED** [review]

**Pull request merged**